socorro/external/crashstorage_base.py

```diff
@@ -589,13 +590,27 @@ def save_processed(self, processed_crash):
     #--------------------------------------------------------------------------
     def save_raw_and_processed(self, raw_crash, dump, processed_crash,
                                crash_id):
+        storage_exception = PolyStorageError()
```

Comment:
I'm not too sure about this, it would make the code overly complicated to avoid create a single object that doesn't take much memory nor processing time.